functions/go/search-replace/metadata.yaml

```diff
@@ -0,0 +1,14 @@
+image: gcr.io/kpt-fn/search-replace
```

Comment:
Given this, I'm assuming you no longer will maintain a list of curated and contrib functions in the Makefile?

Reply:
Yes, we can achieve it with some changes to the scripts and Makefile